scripts/cassandra-integration-test.sh

```diff
@@ -48,14 +52,16 @@ run_integration_test() {
   local major_version=${version%%.*}
   local schema_version=$2
   local jaegerVersion=$3
+  local username="username"
+  local password="password"
```

Comment:
why does this function need to change? If you need to reuse these values, make them global vars. If you don't need to reuse them then just pass constant strings in the one place where they are needed?

Reply:
I added username and password as vars in cassandra-integration-test just to align with how keyspaces are managed, so even the keyspace can be passed as constant string to the apply-schema() but it is done as vars. 
 Another case if it is decided to pass username and password from the main() as user input or from the CI, this would be great start for that.